admin-dev/themes/new-theme/js/components/grid/extension/submit-grid-action-extension.js

```diff
@@ -36,7 +36,7 @@ export default class SubmitGridActionExtension {
   }
 
   extend(grid) {
-    grid.getContainer().on('click', '.js-grid-action-submit-btn', (event) => {
+    grid.getHeaderContainer().on('click', '.js-grid-action-submit-btn', (event) => {
```

Comment:
is it a BC compared to 1.7.5?

How about existing grids in the community modules? /c @sarjon

Reply:
> is it a BC compared to 1.7.5?

Even if it is, it's part of 1.7.6 now.

I see that in 1.7.5 grid actions used to be inside container (retrieved by `getContainer()`) see https://prnt.sc/og9h0z, but in 1.7.6 actions where move outside container, see https://prnt.sc/og9hbv.

It seems wrong to have grid actions outside container. :confused:

Reply:
> I see that in 1.7.5 grid actions used to be inside container (retrieved by `getContainer()`) see https://prnt.sc/og9h0z, but in 1.7.6 actions where move outside container, see https://prnt.sc/og9hbv.

Indeed that's the reason behind this fix, and I agree with

> It seems wrong to have grid actions outside container. 😕

But it's now released in 1.7.6.0 😥

Reply:
Hmmm, I wonder what if we increase the scope of container to the element which is above to cover grid actions? In theory, everything should still be working as original container is still inside?

Reply:
The change happened in https://github.com/PrestaShop/PrestaShop/pull/11095

If existing grids built in 1.7.5 have been using
```
        {% include '@PrestaShop/Admin/Common/Grid/grid_panel.html.twig' with { 'grid': grid } %}
```
then actions have been moved correctly, so using the container header is fine
Thenn

Reply:
It's not a BC since it select only the parent container. It's also working like before if the grid is inside the gridContainer